### ISSUE TITLE
Lacks the 'cancel' button when import opml

### ISSUE THREAD

**Issue by serrq** [jocmp/capyreader#149]
![1000114213](https://github.com/jocmp/capyreader/assets/113377431/9234eabf-80a7-4d63-bfbe-192be723ce3b)

**Comment by jocmp** [comment]
Correct, right now it's all or nothing. I'd like to build out this functionality in the future, but for now it's minimal viable product.

**Comment by jocmp** [comment]
Added, this will be included in the next release

![image](https://github.com/jocmp/capyreader/assets/9521010/ded5330c-de16-4327-ae05-4e086d459d96)
